### PR TITLE
Allows the warden to open the infirmary.

### PR DESCRIPTION
**Pull request by bruhlookatthisdood**
# Document the changes in your pull request

Changed variables so anyone with armory access can open the brig infirmary.
(HoS and warden round start)
((PS sorry for the sloppy code, it was rushed))
# Wiki Documentation

None needed

# Changelog

:cl:  
tweak: Changed brig infimary access so HoS and Warden can open them (anyone with armory access)
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Going to leave this here for people because it's basically the same PR so all discussion on #12209 would apply to this one 
---> #12209 <---

**Comment by bruhlookatthisdood** [comment]
By the way, just want to point out the Chain of Command on the wiki says the warden has Control over the entire brig. The brig infirmary uses the brig APC and is labeled as brig, just a medical section. Therefore, the warden should be able to walk in there.

File: code/modules/jobs/job_types/warden.dm [hunk 1]
```diff
@@ -18,8 +18,8 @@
 
 	alt_titles = list("Brig Watchman", "Brig Superintendent", "Security Staff Sergeant", "Security Dispatcher", "Prison Supervisor")
 
-	access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_ARMORY, ACCESS_COURT, ACCESS_MECH_SECURITY, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_WEAPONS, ACCESS_FORENSICS_LOCKERS, ACCESS_MINERAL_STOREROOM)
-	minimal_access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_ARMORY, ACCESS_MECH_SECURITY, ACCESS_COURT, ACCESS_WEAPONS, ACCESS_MINERAL_STOREROOM) // See /datum/job/warden/get_access()
+	access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_ARMORY, ACCESS_COURT, ACCESS_MECH_SECURITY, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_WEAPONS, ACCESS_FORENSICS_LOCKERS, ACCESS_MINERAL_STOREROOM, ACCESS_BRIG_PHYS)
```
Comment (adamsong) [inline code/modules/jobs/job_types/warden.dm:21]: The doors are ACCESS_MEDICAL not ACCESS_BRIG_PHYS, which is only the locker.

**Review by adamsong — CHANGES_REQUESTED** [review]

**Comment by JamieD1** [comment]
> By the way, just want to point out the Chain of Command on the wiki says the warden has Control over the entire brig. The brig infirmary uses the brig APC and is labeled as brig, just a medical section. Therefore, the warden should be able to walk in there.

With that argument the CMO should have access to the security checkpoint and everything inside it.

**Comment by JamieD1** [comment]
Alternative in progress.

**Pull request closed without merging**